src/components/Capacity.svelte

```diff
@@ -49,20 +54,31 @@
     }
     if (connected && apiPromise?.query && addr) {
       let info = await getMsaEpochAndCapacityInfo(apiPromise, addr);
-      msaInfo = { ...msaInfo, ...info.msaInfo };
+      if (info?.msaInfo) {
+        msaInfo.providerName = providerNameToHuman(info.msaInfo.providerName);
+      }
+      msaInfo.msaId = info?.msaInfo?.msaId || 0;
+      msaInfo.isProvider = info?.msaInfo?.isProvider || false;
       capacityDetails = { ...defaultDetails, ...info.capacityDetails };
       epochNumber = info.epochNumber;
       storeMsaInfo.set(msaInfo);
     }
   });
-  export let token;
 </script>
 
-<div class:hidden={!msaInfo.isProvider}>
-  <h3>Capacity at Block {blockNumber}, Epoch {epochNumber}</h3>
-  <p>Provider name: {msaInfo.providerName}</p>
-  <p><strong>Remaining:</strong> {capacityDetails?.remainingCapacity}</p>
-  <p><strong>Total Issued:</strong> {capacityDetails?.totalCapacityIssued}</p>
-  <p><strong>Last replenished:</strong> Epoch {capacityDetails?.lastReplenishedEpoch}</p>
-  <p><strong>Staked Token:</strong> {capacityDetails?.totalCapacityIssued} {token}</p>
+<div class="pl-6 ml-6 border-l-8 border-aqua">
+  <h3 class="text-aqua font-bold">Capacity</h3>
+  {#if !connected}
+    <p>Not connected</p>
```

Comment:
All the "status" items now display more or less the same information until you are connected and we know if you are a Provider or you just have an MSA.